puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_plugin_matrix.yaml

```diff
@@ -28,7 +28,7 @@
           description: "Git branch name of Foreman itself, e.g. <pre>develop</pre>"
       - string:
           name: dbs
-          default: 'postgresql mysql sqlite3'
+          default:
```

Comment:
It just seemed unnecessary, but I will leave it alone since it does not change anything.